kuma/core/utils.py

```diff
@@ -284,3 +286,11 @@ def chord_flow(pre_task, tasks, post_task):
         return chain(*tasks)
     else:
         return chain(pre_task, chord(header=tasks, body=post_task))
+
+
+def limit_banned_ip_to_0(group, request):
+    try:
+        IPBan.objects.get(ip=get_ip(request))
```

Comment:
If this ever gets applied to more-frequent endpoints, I'd put some layer of caching here.